### PR TITLE
feat(hedera): add memo capability

### PR DESCRIPTION
**Pull request by jackthta**
### 📝 Description

This PR will add the memo capability to Hedera.

### ❓ Context

- **Impacted projects**: `Ledger Live Common & Ledger Live Desktop` 
- **Linked resource(s)**: `N/A` 

### ✅ Checklist

- [X] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 📸 Demo
Screen Shot 2022-06-28 at 11 51 18
Screen Shot 2022-06-28 at 11 51 26
Screen Shot 2022-06-28 at 11 51 37

IMG_0609-min

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by aelmilligy-ledger** [comment]
Hi @jackthta,
Could you have a look on the lint, some checks were not successful because of it. Also, could you include changesets to this PR?

**Comment by haammar-ledger** [comment]
@aelmilligy-ledger Note: this PR could be reviewed first then targeted to `feat/hedera` instead of `develop`, to also run the bot on these changes.

**Comment by haammar-ledger** [comment]
@jackthta Can you please add mutations to the bot's spec file, to test transactions that include a memo?

File: apps/ledger-live-desktop/scripts/sync-families-dispatch.sh [hunk 1]
```diff
@@ -17,7 +17,9 @@ AccountBalanceSummaryFooter.js \
 TokenList.js \
 AccountHeaderManageActions.js \
 StepReceiveFunds.js \
-NoAssociatedAccounts.js
+NoAssociatedAccounts.js \
+StepAmount.js \
```
[the remark this answers is not in the document]
Comment (jackthta) [reply apps/ledger-live-desktop/scripts/sync-families-dispatch.sh:21]: Will do 👍

**Comment by aelmilligy-ledger** [comment]
Hi @jackthta,
Regarding the checks made after your commits, there are still some linter issues. Also, please consider to rebase or merge in order to ensure that your branch is up-to-date.

**Comment by jackthta** [comment]
> Hi @jackthta, Regarding the checks made after your commits, there are still some linter issues. Also, please consider to rebase or merge in order to ensure that your branch is up-to-date.

Hi @aelmilligy-ledger,
Sorry about that, I believe it should be good now, but please do let me know if otherwise.

File: apps/ledger-live-desktop/src/renderer/families/hedera/types.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  FeeStrategy,
+} from "@ledgerhq/live-common/lib/types";
+
+export type SendAmountProps = {
+  account: Account,
+  transaction: Transaction,
+  status: TransactionStatus,
+  onChange: Transaction => void,
+  updateTransaction: (updater: any) => void,
+  mapStrategies?: FeeStrategy => FeeStrategy & { [string]: * },
```
Comment (haammar-ledger) [inline apps/ledger-live-desktop/src/renderer/families/hedera/types.js:16]: Very minor, but these 2 props seem unused, and this is causing a failure in that CI test: https://github.com/LedgerHQ/ledger-live/runs/7099372918?check_suite_focus=true

**Review by haammar-ledger — CHANGES_REQUESTED** [review]

**Comment by henrily-ledger** [comment]
Thanks @jackthta we did take the PR and add the missing small fix, we will close it since we cherry-pick commits and merge it to feat/hedera

**Pull request closed without merging**